mac/Makefile

```diff
@@ -13,12 +15,17 @@ libwooting-rgb-sdk.dylib: $(OBJS)
 	$(CC) $(LDFLAGS) $(LIBS) -shared -fPIC -Wl,-install_name,$0.0 $^ -o $@
 
 $(OBJS): %.o: %.c
-	$(CC) $(CFLAGS) -c $(INCLUDES) $< -o $@
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $(INCLUDES) $< -o $@
 
 clean:
 	rm -f $(OBJS)
 
 install: libwooting-rgb-sdk.dylib
-	cp libwooting-rgb-sdk.dylib /usr/local/lib/libwooting-rgb-sdk.dylib
+	mkdir -p $(prefix)/lib
```

Comment:
Is it possible that this folder doesn't exist on MacOS?

Reply:
I don't know, I wouldn't bet that it's impossible. So I added it just to be sure. If it already exists, it will make no difference.
`/usr/local` should already exist, but I don't know about `lib/`, `include/`, and others, at that path.

Reply:
I missed another point. `prefix` is `prefix`, not `/usr/local`; We can't assume that `prefix` stays `/usr/local`, that's why we made it variable after all…